testutil/keeper/tokenomics.go

```diff
@@ -508,3 +513,24 @@ func WithProposerAddr(addr string) TokenomicsModuleKeepersOpt {
 		return sdkCtx
 	}
 }
+
+// WithProofRequirement is an option to set the proof requirement in the tokenomics module keepers.
+func WithProofRequirement(required bool) TokenomicsModuleKeepersOpt {
+	return func(ctx context.Context, keepers *TokenomicsModuleKeepers) context.Context {
+
+		proofParams := keepers.ProofKeeper.GetParams(ctx)
+		if required {
+			proofParams.ProofRequestProbability = 1
+			proofRequirementThreshold := cosmostypes.NewInt64Coin(volatile.DenomuPOKT, 0)
+			proofParams.ProofRequirementThreshold = &proofRequirementThreshold
+		} else {
+			proofParams.ProofRequestProbability = 0
+			proofRequirementThreshold := cosmostypes.NewInt64Coin(volatile.DenomuPOKT, math.MaxInt64)
+			proofParams.ProofRequirementThreshold = &proofRequirementThreshold
+		}
+
+		keepers.ProofKeeper.SetParams(ctx, proofParams)
```

Comment:
I see two options here:

1. Update `TokenomicsModuleKeepersOpt` to receive a `*testing.T` such that you can call `require.NoError(t, err)` here.
2. `panic(err)`; if it's going to panic, it'll happen on setup.